### ISSUE TITLE
Reverse button

### ISSUE THREAD

**Issue by mxmilkiib** [brycec/vizlamb#2]
Undoing the reduce would be handy for comparison.

**Comment by brycec** [comment]
Cool idea! I like it. I was imagining a timeline of sorts, where you can see forward and backwards a few steps, smaller/faded or something. But we'll definitely need a reverse button.

**Comment by mxmilkiib** [comment]
Sweet, and thanks very much for creating this btw, an interesting framing. Seen the Up and Down the Ladder of Abstraction article by Bret Victor? Also possible; descriptions similar to those on http://vivid.chengyichao.info/

**Comment by brycec** [comment]
Yes! I hope using d3 trees makes different visualizations easier.

I like Vivid Schemer. Strangely similar. Is the source available?

**Comment by mxmilkiib** [comment]
See [onesuper](https://github.com/onesuper). Was also posted on [HN](http://news.ycombinator.com/item?id=5230404).
